build.gradle

```diff
@@ -738,34 +731,21 @@ task distDocker {
   def dockerBuildDir = "build/docker-besu/"
 
   doLast {
-    for (def jvmVariant in dockerVariants) {
-      copy {
-        from file("${projectDir}/docker/${jvmVariant}/Dockerfile")
-        into(dockerBuildDir)
-      }
-      exec {
-        def image = "${dockerImageName}:${dockerBuildVersion}-${jvmVariant}"
-        def dockerPlatform = ""
-        if (project.hasProperty('docker-platform')){
-          dockerPlatform = "--platform ${project.getProperty('docker-platform')}"
-          println "Building for platform ${project.getProperty('docker-platform')}"
-        }
-        def gitDetails = getGitCommitDetails(10)
-        executable shell
-        workingDir dockerBuildDir
-        args "-c", "docker build ${dockerPlatform} --build-arg BUILD_DATE=${buildTime()} --build-arg VERSION=${dockerBuildVersion} --build-arg VCS_REF=${gitDetails.hash} -t ${image} ."
-      }
-    }
-
-    // tag the "default" (which is the variant in the zero position)
-    exec {
-      executable shell
-      args "-c", "docker tag '${dockerImageName}:${dockerBuildVersion}-${dockerVariants[0]}' '${dockerImageName}:${dockerBuildVersion}'"
+    copy {
+      from file("${projectDir}/docker/Dockerfile")
+      into(dockerBuildDir)
     }
-    // create a static tag for the benchmark target
     exec {
+      def image = "${dockerImageName}:${dockerBuildVersion}"
+      def dockerPlatform = ""
+      if (project.hasProperty('docker-platform')){
+        dockerPlatform = "--platform ${project.getProperty('docker-platform')}"
+        println "Building for platform ${project.getProperty('docker-platform')}"
+      }
+      def gitDetails = getGitCommitDetails(10)
```

Comment:
iirc there was a commit from @siladu to use `7` letters for version, should it apply to above as well?